src/romaine/core.py

```diff
@@ -1,6 +1,45 @@
+import os
+
+
 class Core(object):
+    """
+        The core of the Romaine, provides BDD test API.
+    """
+    # All located features
+    features = []
```

Comment:
Maybe actually this should be `feature_file_paths` - we can have `features` as a list of feature objects from the gherkin parser